lib/health_cards/health_card_constraints.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'base64'
+require 'zlib'
+require 'json/minify'
+
+module HealthCards
+  ## Functionality for "Health Cards are Small" section of the Smart Health Cards Specification
+  module HealthCardConstraints
```

Comment:
Is there a reason this is done as a module? Wouldn't this be better as a part of VerifiableCredential?